lib/index.js

```diff
@@ -1,29 +1,25 @@
-
 const createSymbiote = (initialState, actionsConfig, actionTypePrefix = '') => {
   const handlersList = {}
 
-  const traverseActions = (rootConfig, rootPath = []) => {
-    const actionsList = {}
-
-    Object.keys(rootConfig).forEach((key) => {
+  const traverseActions = (rootConfig, rootPath = []) => (
+    Object.keys(rootConfig).reduce((acc, key) => {
       const currentPath = rootPath.concat(key)
       const handler = rootConfig[key]
 
       if (typeof handler === 'function') {
         const type = currentPath.join('/')
 
-        actionsList[key] = (...args) => ({ type, payload: args })
-        actionsList[key].toString = () => type
+        acc[key] = (...args) => ({ type, payload: args })
```

Comment:
rename back to `actionsList` or `actionsAccum`

Reply:
@betula ?

Reply:
dolbaeb